doc/proposals/samples.md

```diff
@@ -0,0 +1,142 @@
+- The project should have an small section over how to test and with links for reference
+- The tests should be done using the default go testing lib and the test-framework provided by SDK
+ 
+**NOTES** 
+- See [here](https://github.com/dev4devs-com/postgresql-operator/blob/master/pkg/controller/database/controller_test.go) an example over how to cover the controller.
+- See [here](https://github.com/dev4devs-com/postgresql-operator/blob/master/pkg/controller/database/fakeclient_test.go) an example over how to create the fake client. 
+
+#### e2e tests for Go Memcached Sample
+
+- I as a Go dev operator user, I'd like to know how to cover the projects with integration tests using the test-framework
+
+**Acceptance Criteria** 
+- The GO project should be covered with a few basic integration tests
+- The tests should be working successfully
+- The project should have a makefile target to call the integration test
+- The project should have an small section over how to test and with links for reference
+- The tests should be done using the default go testing lib and the test-framework provided by SDK
+
+#### Unit tests for Ansible Memcached Sample
+
+- I as a Ansible dev operator user, I'd like to know how to cover the projects with unit tests using molecule
+
+**Acceptance Criteria** 
+- The Ansible project should be covered by tests using [molecule](https://github.com/operator-framework/operator-sdk-samples/tree/master/ansible/memcached-operator/molecule) which by default is scaffold
+- The tests should be working successfully
+- The project should have an small section over how to test and with links for reference
+- The project should have a makefile target to call the tests
+
+#### tests for Helm Memcached Sample
+
+- I as a Helm dev operator user, I'd like to how to cover the projects with tests
+
+**Acceptance Criteria** 
+- The Helm project should have a few tests as using shell as it done [here](https://github.com/operator-framework/operator-sdk/blob/master/hack/tests/e2e-helm.sh).
+- The tests should be working successfully
+- The project should have an small section over how to test and with links for reference
+- The project should have a makefile target to call the tests
+
+
+#### e2e tests for Ansible Memcached Sample
+
+- I as a Ansible dev operator user, I'd like to how to cover the projects with integration tests using molecule
+
+**Acceptance Criteria** 
+- The Ansible project should have a few integration tests using [molecule](https://github.com/operator-framework/operator-sdk-samples/tree/master/ansible/memcached-operator/molecule) which by default is scaffold
+- The tests should be working successfully
+- The project should have an small section over how to test and with links for reference
+- The project should have a makefile target to call the tests
+
+#### Travis Integration
+
+- I as a maintainer, I'd like to ensure that the samples still working after the changes performed in the PR easily
```

Comment:
* maintainer of what? please clarify in the statement.
* Change to: `As a maintener INSERT WHAT, I would like ...`
* rephrase `ensure that samples continue to work after the changes performed in the PR are applied.`